internal/slicer/slicer_test.go

```diff
@@ -53,294 +53,292 @@ var packageEntries = map[string][]testutil.TarEntry{
-		"/etc/dir/sub/":  "dir 01777",
-		"/etc/passwd":    "file 0644 5b41362b",
+		"/dir/":           "dir 0755",
+		"/dir/file":       "file 0644 cc55e2ec",
+		"/dir/file_copy":  "file 0644 cc55e2ec",
+		"/dir/foo/":       "dir 0755",
+		"/dir/foo/bar/":   "dir 01777",
+		"/dir/text_file":  "file 0644 5b41362b",
+		"/other_dir/":     "dir 0755",
+		"/other_dir/file": "symlink ../dir/file",
 	},
 }, {
 	summary: "Glob extraction",
-	slices:  []setup.SliceKey{{"base-files", "myslice"}},
+	slices:  []setup.SliceKey{{"test-package", "myslice"}},
 	release: map[string]string{
-		"slices/mydir/base-files.yaml": `
-			package: base-files
+		"slices/mydir/test-package.yaml": `
+			package: test-package
 			slices:
 				myslice:
 					contents:
-						/**/he*o:
+						/**/other_f*e:
 		`,
 	},
 	result: map[string]string{
-		"/usr/":          "dir 0755",
-		"/usr/bin/":      "dir 0755",
-		"/usr/bin/hello": "file 0775 eaf29575",
+		"/dir/":                  "dir 0755",
+		"/dir/nested/":           "dir 0755",
+		"/dir/nested/other_file": "file 0644 6b86b273",
+		"/dir/other_file":        "file 0644 63d5dd49",
 	},
 }, {
-	summary: "Create new file under extracted directory",
-	slices:  []setup.SliceKey{{"base-files", "myslice"}},
+	summary: "Create new file under extracted directory and preserve parent directory permissions",
+	slices:  []setup.SliceKey{{"test-package", "myslice"}},
 	release: map[string]string{
-		"slices/mydir/base-files.yaml": `
-			package: base-files
+		"slices/mydir/test-package.yaml": `
+			package: test-package
 			slices:
 				myslice:
 					contents:
-						# Note the missing /tmp/ here.
-						/tmp/new: {text: data1}
+						# Note the missing parent directories here.
+						/parent/new: {text: data1}
 		`,
 	},
 	result: map[string]string{
-		"/tmp/":    "dir 01777", // This is the magic.
-		"/tmp/new": "file 0644 5b41362b",
+		"/parent/":    "dir 01777",
+		"/parent/new": "file 0644 5b41362b",
 	},
 }, {
-	summary: "Create new nested file under extracted directory",
-	slices:  []setup.SliceKey{{"base-files", "myslice"}},
+	summary: "Create new nested file under extracted directory and preserve parent directory permissions",
+	slices:  []setup.SliceKey{{"test-package", "myslice"}},
 	release: map[string]string{
-		"slices/mydir/base-files.yaml": `
-			package: base-files
+		"slices/mydir/test-package.yaml": `
+			package: test-package
 			slices:
 				myslice:
 					contents:
-						# Note the missing /tmp/ here.
-						/tmp/new/sub: {text: data1}
+						# Note the missing parent directories here.
+						/parent/permissions/new: {text: data1}
 		`,
 	},
 	result: map[string]string{
-		"/tmp/":        "dir 01777", // This is the magic.
-		"/tmp/new/":    "dir 0755",
-		"/tmp/new/sub": "file 0644 5b41362b",
+		"/parent/":                "dir 01777",
+		"/parent/permissions/":    "dir 0764",
+		"/parent/permissions/new": "file 0644 5b41362b",
 	},
 }, {
 	summary: "Create new directory under extracted directory",
```

Comment:
Why hasn't the summary been changed on this one, when it's a follow up from the ones above?

Reply:
You are right, changed it!